packages/app/src/DOMPlayer.tsx

```diff
@@ -0,0 +1,528 @@
+      searchParams.push({ key, value });
+    }
+  } catch (e) {
+    // ignore
+  }
+
+  return (
+    <HoverCard shadow="md" position="bottom-start">
+      <HoverCard.Target>
+        <div className={styles.playerHeaderUrl}>{url || 'Session Player'}</div>
+      </HoverCard.Target>
+      {url && (
+        <HoverCard.Dropdown>
+          <table className="table fs-8 mb-0">
+            <tr>
+              <td>
+                <i className="bi bi-globe fs-8 text-slate-300"></i>
+              </td>
+              <td>{parsedUrl?.host}</td>
+            </tr>
+            <tr>
+              <td>
+                <i className="bi bi-link-45deg text-slate-300 fs-7"></i>
+              </td>
+              <td>{parsedUrl?.pathname}</td>
+            </tr>
+            {searchParams &&
+              searchParams.length > 0 &&
+              searchParams.map(param => (
+                <tr key={param.key}>
+                  <td>
+                    <strong>{param.key}</strong>
+                  </td>
+                  <td>{param.value}</td>
+                </tr>
+              ))}
+          </table>
+        </HoverCard.Dropdown>
+      )}
+    </HoverCard>
+  );
+});
+
+export default function DOMPlayer({
+  config: { sourceId, dateRange, sessionId },
+  focus,
+  setPlayerTime,
+  playerState,
+  setPlayerState,
+  playerSpeed,
+  skipInactive,
+  setPlayerStartTimestamp,
+  setPlayerEndTimestamp,
+  setPlayerFullWidth,
+  playerFullWidth,
+  resizeKey,
+}: {
+  config: {
+    sourceId: string;
+    sessionId: string;
+    dateRange: [Date, Date];
+  };
+  focus: { ts: number; setBy: string } | undefined;
+  setPlayerTime: (ts: number) => void;
+  playerState: 'playing' | 'paused';
+  setPlayerState: (state: 'playing' | 'paused') => void;
+  playerSpeed: number;
+  setPlayerStartTimestamp?: (ts: number) => void;
+  setPlayerEndTimestamp?: (ts: number) => void;
+  // setPlayerSpeed: (playerSpeed: number) => void;
+  skipInactive: boolean;
+  // setSkipInactive: (skipInactive: boolean) => void;
+  // highlightedResultId: string | undefined;
+  // onClick: (logId: string, sortKey: number) => void;
+  resizeKey?: string;
+  setPlayerFullWidth: (fullWidth: boolean) => void;
+  playerFullWidth: boolean;
+}) {
+  const debug = useDebugMode();
+  const wrapper = useRef<HTMLDivElement>(null);
+  const playerContainer = useRef<HTMLDivElement>(null);
+  const replayer = useRef<Replayer | null>(null);
+  const initialEvents = useRef<any[]>([]);
+
+  const lastEventTsLoadedRef = useRef(0);
+  const [lastEventTsLoaded, _setLastEventTsLoaded] = useState(0);
+  const setLastEventTsLoaded = useRef(
+    throttle(_setLastEventTsLoaded, 100, { leading: true, trailing: true }),
+  );
+  const [isInitialEventsLoaded, setIsInitialEventsLoaded] = useState(false);
+  const [isReplayFullyLoaded, setIsReplayFullyLoaded] = useState(false);
+
+  let currentRrwebEvent = '';
+
+  const { isFetching: isSearchResultsFetching, abort } = useRRWebEventStream(
```

Comment:
`useRRWebEventStream` is mostly copied over with a bit of modifications that talk to sessions endpoint through the proxy